addons/hw_posbox_homepage/static/src/app/components/dialog/UpdateDialog.js

```diff
@@ -25,66 +25,95 @@ export class UpdateDialog extends Component {
+        <BootstrapDialog identifier="'update-configuration'" btnName="'Update'" onOpen.bind="getVersionInfo" onClose.bind="onClose">
             <t t-set-slot="header">
-                Upgrade IoTBox
+                <div>
+                    Update IoT Box
+                    <a href="https://www.odoo.com/documentation/18.0/applications/general/iot/config/updating_iot.html" class="fa fa-question-circle text-decoration-none text-dark" target="_blank"></a>
+                </div>
             </t>
             <t t-set-slot="body">
                 <div t-if="this.state.initialization" class="position-absolute top-0 start-0 bg-white h-100 w-100 justify-content-center align-items-center d-flex flex-column gap-3" style="z-index: 9999">
                     <div class="spinner-border" role="status">
                         <span class="visually-hidden">Loading...</span>
                     </div>
-                    <p>Currently fetching upgrade data...</p>
+                    <p>Currently fetching update data...</p>
                 </div>
 
-                <div class="alert alert-warning fs-6" role="alert">
-                    This tool will help you perform an upgrade of the IoTBox's software over the internet.
-                    However the preferred method to upgrade the IoTBox is to flash the sd-card with the latest image.
-                    The upgrade procedure is explained into to the IoTBox manual.<br/><br/>
-                    Usefull links:
-                    <ul>
-                        <li>
-                            <a href="https://nightly.odoo.com/master/iotbox/iotbox-latest.zip">Download the latest image</a>
-                        </li>
-                        <li>
-                            <a href="https://www.odoo.com/documentation/18.0/applications/productivity/iot.html">IoTBox manual</a>
-                        </li>
-                    </ul>
+                <h6>System Image</h6>
+                <div t-if="this.state.imageIsUpToDate" class="alert alert-success">
+                    <h7>System is up to date! (<t t-esc="this.store.base.version"/>)</h7>
+                </div>
+                <div t-else="" class="alert alert-warning">
+                    <h7>A new image is available!</h7>
+                    <div>
+                        See:
+                        <a href="https://www.odoo.com/documentation/18.0/applications/general/iot/config/updating_iot.html#flashing-the-sd-card-on-iot-box" target="_blank" class="alert-link">
+                            Flashing the SD Card on IoT Box
+                        </a>
+                    </div>
                 </div>
 
-                <div class="bg-light rounded p-2 fs-6">
-                    <h6>Commit details:</h6>
-                    <pre t-out="this.state.commitHtml" />
+                <h6>Odoo Service</h6>
```

Comment:
can you rename it to "IoT core code" ? Except if the "Odoo Service" code is the new official name